utils.sh

```diff
@@ -211,7 +211,7 @@ function patch_ep_host_etcd() {
     declare -r domain="$1"
     declare -r srv_record="_etcd-server-ssl._tcp.$domain"
     declare -r api_domain="api.$domain"
-    host_etcd_ep=$(oc get ep -n kube-system host-etcd -o json | jq -r '{"subsets": .subsets}')
+    host_etcd_ep=$(oc get ep -n openshift-etcd host-etcd -o json | jq -r '{"subsets": .subsets}')
```

Comment:
```console
[root@dell-r730-021 dev-scripts]# oc get ep -n kube-system host-etcd -o json | jq -r '{"subsets": .subsets}'
Error from server (NotFound): endpoints "host-etcd" not found

[root@dell-r730-021 dev-scripts]# oc get ep -n openshift-etcd host-etcd -o json | jq -r '{"subsets": .subsets}'
{
  "subsets": [
    {
      "addresses": [
        {
          "hostname": "etcd-0",
          "ip": "192.168.111.20"
        },
        {
          "hostname": "etcd-1",
          "ip": "192.168.111.21"
        },
        {
          "hostname": "etcd-2",
          "ip": "192.168.111.22"
        }
      ],
      "ports": [
        {
          "name": "etcd",
          "port": 2379,
          "protocol": "TCP"
        }
      ]
    }
  ]
}
```

Reply:
openshift/installer#1665 is a good reference on the background for this change